source/interprocedural_analyses/taint/test/configurationTest.ml

```diff
@@ -1594,53 +1609,53 @@ let test_matching_kinds _ =
     ~transform_filter:["TU"; "TW"]
     ~configuration:
       {|
-        { sources: [
-            { name: "A" },
-            { name: "B", kind: "parametric" },
-            { name: "C" },
-            { name: "D" }
+        { "sources": [
+            { "name": "A" },
+            { "name": "B", "kind": "parametric" },
+            { "name": "C" },
+            { "name": "D" }
           ],
-          sinks: [
-            { name: "X" },
-            { name: "Y", kind: "parametric" },
-            { name: "Z" }
+          "sinks": [
+            { "name": "X" },
+            { "name": "Y", "kind": "parametric" },
+            { "name": "Z" }
           ],
-          transforms: [
-            { name: "TU" },
-            { name: "TV" },
-            { name: "TW" }
+          "transforms": [
+            { "name": "TU" },
+            { "name": "TV" },
+            { "name": "TW" }
           ],
-          rules: [
+          "rules": [
             {
-               name: "test rule",
-               sources: ["A", "D"],
-               sinks: ["X", "Y"],
-               code: 1,
-               message_format: ""
+               "name": "test rule",
+               "sources": ["A", "D"],
+               "sinks": ["X", "Y"],
+               "code": 1,
+               "message_format": ""
             },
             {
-               name: "test rule 2",
-               sources: ["B"],
-               transforms: ["TU"],
-               sinks: ["Y"],
-               code: 2,
-               message_format: ""
+               "name": "test rule 2",
+               "sources": ["B"],
+               "transforms": ["TU"],
+               "sinks": ["Y"],
+               "code": 2,
+               "message_format": ""
             },
             {
-               name: "test rule 3",
-               sources: ["C"],
-               transforms: ["TV"],
-               sinks: ["Z"],
-               code: 3,
-               message_format: ""
+               "name": "test rule 3",
+               "sources": ["C"],
+               "transforms": ["TV"],
+               "sinks": ["Z"],
+               "code": 3,
+               "message_format": ""
             },
             {
-               name: "test rule 4",
-               sources: ["C"],
-               transforms: ["TU", "TW"],
-               sinks: ["Z"],
-               code: 4,
-               message_format: ""
+               "name": "test rule 4",
+               "sources": ["C"],
+               "transforms": ["TU", "TW"],
+               "sinks": ["Z"],
+               "code": 4,
+               "message_format": ""
```

Comment:
Could we move all these changes in a different commit/PR? It's not strictly related to that PR although I understand why it's necessary.

I guess Github Pull Requests make it difficult to have mutltiple PRs depending on each other.

Reply:
Another possible problem is that one requires the other to make sure all tests pass - I think it's not recommended to split in that scenario.